src/files.js

```diff
@@ -374,6 +374,7 @@ FOAM_FILES([
   { name: "foam/parse/QueryParser" },
   { name: "foam/parse/FScript" },
   { name: "foam/parse/QueryParserUserTest" },
+  { name: "foam/parse/QueryParserTest" },
```

Comment:
I think we should be adding:
  , flags: ['debug']
So that we don't include in final builds.
Or, maybe we introduce a new 'test' flag. You can just add it, it won't hurt.